src/main/java/org/opentripplanner/middleware/persistence/Persistence.java

```diff
@@ -74,6 +84,9 @@ public static void initialize () {
         apiUsers = new TypedPersistence(mongoDatabase, ApiUser.class);
         tripRequests = new TypedPersistence(mongoDatabase, TripRequest.class);
         tripSummaries = new TypedPersistence(mongoDatabase, TripSummary.class);
+        bugsnagEventRequests = new TypedPersistence(mongoDatabase, BugsnagEventRequest.class);
+        bugsnagEvents = new TypedPersistence(mongoDatabase, BugsnagEvent.class);
+        bugsnagProjects = new TypedPersistence(mongoDatabase, BugsnagProject.class);
```

Comment:
This approach is the only way I can reliably get event and project information from Bugsnag without hitting API rate limits.

- Event requests are held because they aren't immediately fulfilled.
- Events are held for the dashboard.
- Projects are held so that to name can be provided to the dashboard (The event information only contains the project id).